src/main/java/sirius/db/mixing/types/MultiLanguageString.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.db.mixing.types;
+
+import sirius.kernel.Sirius;
+import sirius.kernel.health.Exceptions;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Provides a language-text map as property value.
+ * <p>
+ * These are stored in MongoDB as an array containing sub-documents
+ * containing a <tt>lang</tt> and <tt>text</tt> properties
+ *
+ * @see sirius.db.mixing.properties.MultiLanguageStringProperty
+ */
+public class MultiLanguageString extends SafeMap<String, String> {
+
+    private static final List<String> supportedLanguages =
+            Sirius.getSettings().getConfig("mongo").getStringList("supportedLanguages");
```

Comment:
this static initializer is evaluated too early (config might not be present yet)
Load on first use(?) Also use a Set in favor of a list(?)

Reply:
Using a Set, lazy-load